themes/src/main/resources/theme/keycloak/login/resources/css/login.css

```
@@ -253,6 +253,10 @@ ul#kc-totp-supported-apps {
     margin-top: 20px;
 }
 
+.kc-social-links > li {
```

I don't expect there will be anything but direct descendants here so we can reduce the specificity here.
```suggestion
.kc-social-links li {
```

---

👍 Done.